client/src/actions/user_signup.js

```diff
@@ -0,0 +1,165 @@
+// User Actions
```

Comment:
Personally I think it makes more sense to separate the User actions from the SignUp actions, since they each have their own separate states in our store. @terencetmac @tylerarbus  What do you guys think?